src/core_modules/capture-core/src/metaData/DataElement/elementTypes.js

```diff
@@ -23,6 +23,7 @@ const elementTypeCodes = [
     'DURATION_MINUTES',
     'FILE_RESOURCE',
     'URL',
+    'ORG_UNIT_TREE',
```

Comment:
the value type for this dataelement is ORGANISATION_UNIT

Reply:
updated